src/styles/utils/variables.scss

```
@@ -0,0 +1,7 @@
+$card-border-color: #f3f3f3;
+$main-text-color: #060b35;
+$product-code-text-color: #616070;
+$price-text-color: #616070;
```

Don't need to duplicate variables with the same values
```suggestion
$product-code-text-color: #616070;
```